.github/workflows/run-tests-on-pr copy.yaml

```diff
@@ -0,0 +1,60 @@
+name: Run Tests on PR Label
```

Comment:
Remove this copy file

Reply:
Keep it but remove ` copy` from filename

Reply:
done